src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/Api/UnitTestingEncodedStringTextHelpers.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.IO;
+using System.Text;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
+{
+    internal static class UnitTestingEncodedStringTextHelpers
```

Comment:
💭 Seems like this is an accessor type

```suggestion
    internal static class UnitTestingEncodedStringTextAccessor
```